src/checkpoints.h

```diff
@@ -19,18 +19,18 @@ struct CBadpointData;
  */
 namespace Checkpoints
 {
-
+    
 //! Returns last CBlockIndex* in mapBlockIndex that is a checkpoint
 CBlockIndex* GetLastCheckpoint(const CCheckpointData& data);
 
 //! Add an in-memory checkpoint
-void AddCheckPoint(const CCheckpointData& data, int64_t height, uint256& hash);
+void AddCheckPoint(const CCheckpointData& data, int64_t height, uint256 hash);
```

Comment:
having uint256 & was causing compile errors - something to do with passing r-values as a reference.